nni/tools/nnictl/updater.py

```diff
@@ -58,8 +58,9 @@ def get_query_type(key):
 
 def update_experiment_profile(args, key, value):
     '''call restful server to update experiment profile'''
-    nni_config = Config(get_config_filename(args))
-    rest_port = nni_config.get_config('restServerPort')
+    experiment_config = Experiments()
+    experiment_dict = experiment_config.get_all_experiments()
+    rest_port = experiment_dict.get(get_config_filename(args), None).get('port', None)
```

Comment:
you changed many places, just curious how to know all the related parts are changed, do we have ut to cover nnictl logic?

Reply:
To test this, we need ut that use `launch from python` launch a experiment, and use nnictl to manage it...

Reply:
do we have ut to cover nnictl?